### PR TITLE
Added LCM user properties (beta endpoint)

### PR DESCRIPTION
**Pull request by cmmdesai**

### CODE REVIEW

File: api-reference/beta/resources/user.md [hunk 1]
```diff
@@ -157,14 +157,18 @@ This resource supports:
...
 | creationType | String | Indicates whether the user account was created as a regular school or work account (`null`), an external account (`Invitation`), a local account for an Azure Active Directory B2C tenant (`LocalAccount`) or self-service sign-up using email verification (`EmailVerified`). <br><br>Returned only on $select. Read-only. |
 | deletedDateTime | DateTimeOffset | The date and time the user was deleted. <br><br>Returned only on $select. |
 | department | String | The name for the department in which the user works. <br><br>Returned only on $select. Supports $filter. |
 | displayName | String | The name displayed in the address book for the user. This value is usually the combination of the user's first name, middle initial, and last name. This property is required when a user is created and it cannot be cleared during updates. <br><br>Returned by default. Supports $filter and $orderby.|
+| division | String | The name of the division in which the user works. <br><br>Returned only on $select. Supports $filter. |
```
Comment (alok-bansal) [inline api-reference/beta/resources/user.md:168]: Remove Division from here.

File: api-reference/beta/resources/user.md [hunk 2]
```diff
@@ -157,14 +157,18 @@ This resource supports:
 | businessPhones | String collection | The telephone numbers for the user. Only one number can be set for this property. <br><br>Returned by default. |
 | city | String | The city in which the user is located. <br><br>Returned only on $select. Supports $filter. |
 | companyName | String | The company name which the user is associated. This property can be useful for describing the company that an external user comes from. The maximum length of the company name is 64 chararcters.<br><br>Returned only on $select. |
+| costCenter | String | The cost center associated with the user. <br><br>Returned only on $select. Supports $filter. |
```
Comment (alok-bansal) [inline api-reference/beta/resources/user.md:160]: Remove cost center and add employeeOrgData with link to md file for that.

**Review by alok-bansal — CHANGES_REQUESTED** [review]

**Comment by cmmdesai** [comment]
#sign-off

**Review by alok-bansal — APPROVED** [review]

**Comment by alok-bansal** [comment]
@microsoftgraph/graph-docs-contributors - can you please review?

**Comment by alok-bansal** [comment]
@davidmu1 - requesting your review.

**Review by davidmu1 — APPROVED** [review]

**Pull request merged**